lib/elixir/lib/kernel.ex

```diff
@@ -6211,9 +6211,8 @@ defmodule Kernel do
 
   """
   @doc since: "1.17.0"
-  @spec to_timeout([component, ...] | timeout() | Duration.t()) :: timeout()
-        when component: [{unit, non_neg_integer()}, ...],
-             unit: :week | :day | :hour | :minute | :second | :millisecond
+  @spec to_timeout([{unit, non_neg_integer()}] | timeout() | Duration.t()) :: timeout()
+        when unit: :week | :day | :hour | :minute | :second | :millisecond
```

Comment:
I inlined it and I also removed `...`. An empty list should be fine and it already means a timeout of zero.